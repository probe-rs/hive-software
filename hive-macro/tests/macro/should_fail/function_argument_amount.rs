#[hive_macro::hive_test]
fn test(arg1: u8, arg2: u8, arg3: u8) {}

fn main() {}
