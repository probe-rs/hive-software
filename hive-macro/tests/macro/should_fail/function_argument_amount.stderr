error: Incorrect amount of function arguments

         = help: Hive testfunctions should have 2 arguments but 3 were supplied.
         = note: example: fn my_fancy_test(test_channel: &mut dyn TestChannelHandle, probe: &mut Probe);

 --> tests/macro/should_fail/function_argument_amount.rs:2:9
  |
2 | fn test(arg1: u8, arg2: u8, arg3: u8) {}
  |         ^^^^
